spec/factories/cars.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :car do
+    user_id { create(:correct_user_update).id }
```

Comment:
```suggestion
    user
```

Reply:
Can I use it here:
association :user, factory: [:user, :correct_user_update]